src/libraries/formatUtils/dataFormatting.ts

```diff
@@ -14,9 +15,15 @@ export const opdDataFormatter = (
     diseases && data.disease3 !== ""
       ? diseases.find((el) => el.code === +data.disease3)
       : null;
-
-  data.date = format(+data.date, "yyyy-MM-dd");
+  data.date = Date.parse(new Date(+data.date).toString()).toString();
   data.visitDate = data.date;
-
   return data;
 };
+
+export const renderDate = (date: string) => {
+  return moment(date).isValid()
+    ? moment(date).format("DD/MM/YYYY")
+    : moment(+date).isValid()
+    ? moment(+date).format("DD/MM/YYYY")
+    : "";
```

Comment:
I think this function could be write in this way:  

```
return moment(date).isValid()
    ? moment(date).format("DD/MM/YYYY")
    : "";
```

Do you agree?

Reply:
To handle timestamp formatting we should use moment(+date).format("format string). So if we want to ignore date string we should use 
`
return moment(+date).isValid()
    ? moment(+date).format("DD/MM/YYYY")
    : "";
`

Reply:
May be we should use return moment(+date).isValid()
    ? moment(+date).format("DD/MM/YYYY")
    : ""; As we are working with timestamp

Reply:
Is this already done?

Reply:
OK, I applied since.